pkg/controller/webhook/mutate_kube_apiserver.go

```diff
@@ -25,6 +25,7 @@ func (m *kubeAPIServerMutator) Default(_ context.Context, obj runtime.Object) er
 		return fmt.Errorf("expected *appsv1.Deployment but got %T", obj)
 	}
 
+	// TODO: This label approach is deprecated and no longer needed in the future. Remove it as soon as gardener/gardener@v1.75 has been released.
```

Comment:
As this webhook is only injecting this label, that means the entire webhook can be removed, right?
Which version of gardener adds the support for the alternative network policy configuration? I mean, I don't have to wait gardener to completely remove support for the deprecated labels/annotations in v1.75, I guess I can remove it even earlier.

Reply:
> As this webhook is only injecting this label, that means the entire webhook can be removed, right?

Yep, clarified it in the TODO statement.

> Which version of gardener adds the support for the alternative network policy configuration?

Probably v1.71 (depends on https://github.com/gardener/gardener/pull/7907).

> I mean, I don't have to wait gardener to completely remove support for the deprecated labels/annotations in v1.75, I guess I can remove it even earlier.

Sure, that's possible. Operators must ensure that at least g/g@v1.71 is deployed when a version of this extension is used that has removed the old deprecated annotations.